src/docs/WSL_BUILD.md

```diff
@@ -0,0 +1,25 @@
+# Local development build [Ubuntu Linux / WSL2]
+
+Make sure you have the dependencies.
+- Install Python 
+```
+sudo apt update
+sudo apt install python3 python3-pip
+```
+- Add the following lines to your ~/.bashrc file: 
```

Comment:
This can also lead to confusion. This is going to change everything you run to be python3 instead of python2. It is a very broad change.

Reply:
sure, except that your build script is specifically looking for the aliased version of python, which doesn't exist unless you add the alias. I can add this detail to the doc so they can decide if it is what they need/want.

Reply:
I don't think the scripts even call python. It is only platformio that does. IIRC, platformio only works in python3. This is another reason I always use virtualenv. Once you set it, it will be python3 whenever you source that activation.

Reply:
I'll test it without the aliases and if it works, I'll remove it from the docs alltogether.